src/components/TwoLevelSelector.js

```diff
@@ -0,0 +1,377 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+const Radium = require('radium');
+
+const Icon = require('./Icon');
+const StyleConstants = require('../constants/Style');
+
+const TwoLevelSelector = React.createClass({
```

Comment:
I'm not a huge fan of this name. Any other suggestions? Here are some options: `FlyoutMenu`, `TieredMenu`